sensor/marker_radar_lidar_calibrator/README.md

```diff
@@ -0,0 +1,136 @@
+# marker_radar_lidar_calibrator
+
+A tutorial for this calibrator can be found [here](../docs/tutorials/marker_radar_lidar_calibrator.md)
+
+## Purpose
+
+The package `marker_radar_lidar_calibrator` allows extrinsic calibration among the radar sensor and lidar sensor used in autonomous driving and robotics.
+
+## Inner-workings / Algorithms
+
+The calibration process involves five steps: constructing a background model, detecting reflectors from sensor data, matching and filtering lidar and radar detections, conducting the calibration, and showing the evaluation result.
+
+### Step 1: Background model extraction (radar & lidar)
+
+Once the calibrator is initiated and the user presses the `Extract background model` button, the calibrator will begin using the lidar pointcloud and radar messages to create background models. These models include voxels that indicate whether they represent the background or not.
```

Comment:
initiated is not correct/clear here

Reply:
fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/baf6119d012f2185437adf9ed11c7badbfe7f9a9